specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerApps.json

```diff
@@ -755,6 +755,21 @@
                 "modelAsString": true
               }
             },
+            "runningStatus": {
+              "description": "Running status of the Container App.",
+              "enum": [
+                "Progressing",
+                "Running",
+                "Stopped",
+                "Suspended"
+              ],
```

Comment:
Although almost intuitive, please add descriptions for each of the enum values.

Reply:
Progressing: the container app is transitioning between stopped and running.
Running: the container app is running.
Stopped: the container app is stopped.